countries/fixtures/default_countries.json

```diff
@@ -215,7 +215,7 @@
 {"pk": 212, "model": "countries.country", "fields": {"code": "SE", "name": "Sweden"}},
 {"pk": 213, "model": "countries.country", "fields": {"code": "CH", "name": "Switzerland"}},
 {"pk": 214, "model": "countries.country", "fields": {"code": "SY", "name": "Syrian Arab Republic"}},
-{"pk": 215, "model": "countries.country", "fields": {"code": "TW", "name": "Taiwan, Province of China"}},
+{"pk": 215, "model": "countries.country", "fields": {"code": "TW", "name": "Taiwan"}},
```

Comment:
Thanks for changing - the country name of Taiwan is correct **without "Province of China"**. The political issue between Taiwan and China is really complicated and could sometimes be really sensitive. We are separated from each other since 1949. As a Taiwanese, I would be happy to see Taiwan stay as it is :)